### ISSUE TITLE
Private reverse DNS servers broken & not serving IPv4 anymore

### ISSUE THREAD

**Issue by trippleflux** [AdguardTeam/AdGuardHome#6282]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, AMD64 (aka x86_64)

### Installation

Custom package (OpenWrt, HomeAssistant, etc; please mention in the description)

### Setup

On one machine

### AdGuard Home version

4479b32ad4d298bc62dce60c32375cc7605c5638

### Action

My OpenWrt Router seems doesn't serve IPv4 anymore after installing 4479b32ad4d298bc62dce60c32375cc7605c5638, only IPv6.

Also getting error when restore my previously backup adguardhome.yaml, need to start adguardhome from scratch, and found out that "Private reverse DNS servers" is broken, cannot set it up with "127.0.0.1:5353".

### Expected result

Adguardhome serving is also serving IPv4

### Actual result

Serving IPv4 is seems to be broken.

**Comment by ainar-g** [comment]
That commit should not interfere with either of these.  Please enable [verbose logs](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#verboselog) to clarify what could be wrong.

> Custom package (OpenWrt, HomeAssistant, etc; please mention in the description)

What kind of custom package are you using exactly?

**Comment by trippleflux** [comment]
Thanks, there are something that I am not quite understand : 
```
2023/10/05 22:41:39.954258 19937#25 [debug] rdns: resolving "10.10.8.142": resolving a private address: exchanging with [::1]:53 over udp: read udp [::1]:52368->[::1]:53: read: connection refused
2023/10/05 22:41:39.954275 19937#25 [debug] clients: finished processing 10.10.8.142 with rdns in 749.881µs
2023/10/05 22:41:39.954288 19937#25 [debug] clients: processing 10.10.8.142 with whois
2023/10/05 22:41:39.954302 19937#25 [debug] clients: finished processing 10.10.8.142 with whois in 13.104µs
```

The 10.10.8.142 is the IP that failed to get automatically an IPv4 from the DHCP/adguardhome but it seems it also using IPv6 DHCP?. While the client (a Windows 11 22H2) is perfectly fine getting automatically IPv6 from adguardhome whine IPv4 suddenly not, before it was both working perfectly fine.

It's on OpenWrt self-compiled latest Snapshot, the snapshot that I have installed currently have problems with logging capabilities (testing out syslog-ng instead of defauly OpenWrt's BusyBox logger).

**Comment by trippleflux** [comment]
Seems fixed with e79981e8bb71d5b44ee2bf329187e72e6662d52b, closing.
